app/assets/javascripts/components/restaurantList.js.jsx

```diff
@@ -55,6 +73,14 @@ class RestaurantList extends React.Component {
           <span>Filter Results:</span>
           <input type="text" placeholder="Filter by name"
                    onChange={this.handleNameFilterChanged.bind(this)} className="filter"/>
+          <select onChange={this.handleCuisineChange.bind(this)} className="filter">
+            <option value={this.allCuisines}>{this.allCuisines}</option>
```

Comment:
I didnt want to hardcode a string. So I kept it in a constant. Should I just put a string such as "All" in the value instead?